README.md

```diff
@@ -23,7 +23,8 @@ huggingface-cli download \
   --local-dir video-dataset-disney
 ```
 
-Then launch LoRA fine-tuning for text-to-video (modify the different hyperparameters, dataset root, and other configuration options as per your choice):
+Then launch LoRA fine-tuning for text-to-video (modify the different hyperparameters, dataset root, and other
```

Comment:
Is there a reason why we've limited the line length? It makes it a bit harder to review all the changes and seems unnecessary because line wrapping doesn't really affect the GitHub README view

Reply:
Oh, this is because I used PyCharm for editing with its automatic formatting (the display for lines exceeding the length might have gone beyond my screen, and I think some developers have the same issue, so I used the IDE's auto-formatting, which might need adjustment